src/Restaurant.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Restaurant {
```

Comment:
Pretty much code from mainLogic should be in Restaurant class and getting info from the user should be in some other class (newly created)